docs/file-reference.md

```diff
@@ -54,6 +56,14 @@ Each service has name and list of the containers.
 
 Name of the service.
 
+#### replicas
+| type  | required |
+|-------|----------|
+| int32 |    no    |
+
+Number of desired pods of this particluar service. This is an optional field.
+If nothing is provided the artifact generated won't have anything either.
```

Comment:
I don't know bout his sentence :-(
It won't say anything useful for people that are not familiar with Kubernetes.
I think that we should avoid mentioning generated artifacts as much as possible.